<launch>
  <node pkg="coordinator" type="main.py" name="coordinator" args="$(find coordinator)/waypoints.csv" output="screen">
  <param name="coordinate_frame" type="string" value="map" />
  </node>

  <node pkg="coordinator" type="world_state.py" name="world_state" output="screen">
  	<param name="use_rosplan" type="boolean" value="False" />
  	<param name="world_config_file" type="string" value="$(find coordinator)/config/lth_robotlab.yaml" />
  </node>
  
  <node pkg="coordinator" type="action_feeder.py" name="action_feeder" args="$(find coordinator)/actionsequences/actions.csv" output="screen">
  </node>
</launch>
